receiver/hostmetricsreceiver/internal/scraper/pagingscraper/pagefile_openbsd.go

```diff
@@ -0,0 +1,102 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build openbsd
```

Comment:
This looks identical to the linux version, with the exception of constants being named differently.  Can we combine them?

Reply:
I'm not sure this implementation is correct - OpenBSD dropped support for the /proc filesystem in 2015, and we should probably be using the swapctl-based implementation from pagefile_bsd.go instead.